### PR TITLE
Feat/issue 27 styling delete

### PR DESCRIPTION
**Pull request by hellodebs**
Hello, 
I didn't to that much but took me a long time. XD
Making the delete button work before adding items is an odd order, I know. Other than that I styled my Order page a little, added images and moved a state.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -10,6 +10,24 @@ import Div100vh from "react-div-100vh";
...
+    },
+    {
+      id: 9,
+      quantity: 3,
+    },
+    {
+      id: 11,
+      quantity: 1,
+    },
+  ]);
```
Comment (patrikglanemann) [inline src/App.js:30]: Good way to test the delete button for now, I like that:)

File: src/App.js [hunk 2]
```diff
@@ -21,6 +39,14 @@ function App() {
       });
   }, []);
 
+  function deleteButtonHandler(id) {
+    const deletedItems = order.filter((deletedItem) => {
+      return id !== deletedItem.id;
+    });
+    setOrder(deletedItems);
+    console.log(deletedItems);
```
Comment (patrikglanemann) [inline src/App.js:47]: I found it, the forbidden console.log XD

File: src/components/MenuItem.js [hunk 3]
```diff
@@ -4,20 +4,22 @@ import { BiMinus, BiPlus } from "react-icons/bi";
...
-        />
-        <p>
-          {item.type}
-          {item.name}
-          {item.description}
-        </p>
+        {image}
+        <div>
+          <p className="menu__item--type">{item.type}</p>
+          <p>{item.name}</p>
```
Comment (patrikglanemann) [inline src/components/MenuItem.js:20]: Maybe this could be a h2 and h3 (for semantic, or SEO)

File: src/components/OrderItem.js [hunk 4]
```diff
@@ -1,17 +1,27 @@
...
+        <img
+          src={`/images/menu/${orderItem.id}.jpg`}
+          alt={orderItem.name}
+          className="order__item--image"
+        />
+        <div>
+          <p>Id: {orderItem.id}</p>
+          <p>Name: {orderItem.name}</p>
+          <p>Quantity: {orderItem.quantity}</p>
+        </div>
```
Comment (patrikglanemann) [inline src/components/OrderItem.js:17]: Same as above, might use header elements instead of p.

File: src/App.js (hunk 2 above)
Comment (hellodebs) [reply src/App.js:47]: HAHAHA i though maybe you wouldnt see it. I saw it after 😄

**Pull request merged**